docs/src/pages.ts

```diff
@@ -208,14 +208,14 @@ const pages: readonly MuiPage[] = [
         pathname: '/api-docs/data-grid',
```

Comment:
```suggestion
        pathname: '/api/data-grid',
```
?

Reply:
Also should line 203 be changed too?

Reply:
I'm not sure that my fix is correct in the first place. Next.js uses the pathname to fetch the .js page file, I might have broken it.

Reply:
My bad, prefetch doesn't work between zones: https://nextjs.org/docs/advanced-features/multi-zones, confirmed in https://github.com/vercel/next.js/issues/4419#issuecomment-390142581